exercise-core/src/main/java/com/examples/attsw/exercise/core/controller/EmployeeController.java

```diff
@@ -0,0 +1,26 @@
+package com.examples.attsw.exercise.core.controller;
+
+import java.util.stream.Collectors;
+
+import com.examples.attsw.exercise.core.service.IEmployeeService;
+
+public class EmployeeController {
+
+	private IEmployeeService employeeService;
+
+	public EmployeeController(IEmployeeService iEmployeeService) {
+		this.employeeService = iEmployeeService;
+	}
+
+	public String getAllEmployee() {
```

Comment:
OOPS... my mistake I called that method getAllEmployee in the original issue (which I now updated): of course this should be getAllEmployees, so please refactor also your code